app/src/ui/branches/branches-container.tsx

```diff
@@ -249,6 +248,11 @@ export class BranchesContainer extends React.Component<
     if (isRefInThisRepo) {
       this.checkoutBranch(head.ref)
     } else {
+      log.debug(
+        `onPullRequestClicked, but we can't checkout the branch: '${
```

Comment:
This was just for my sanity while testing. It shouldn't be necessary any more once #3602 lands.